src/odemis/gui/comp/viewport.py

```diff
@@ -1247,7 +1394,7 @@ def __init__(self, *args, **kwargs):
         """
         # Call parent constructor at the end, because it needs the legend panel
         super(SpatialSpectrumViewport, self).__init__(*args, **kwargs)
-        self.stream = None
+        self.stream = None  # self.stream is either the DataProjection or a Stream which does its own projection
```

Comment:
Eventually, it should be possible to call that `self.proj`, and rename `connect_stream()` -> `connect_proj()`, but let's keep it for later.